mpu/mpu.go

```diff
@@ -32,13 +32,21 @@ import (
 )
 
 // Current version.
-var PackageVersion = "v0.1"
+var PackageVersion = "v0.2b"
 
 // InitMPU
 func InitMPU(sample_rate, yaw_mix_factor int) int {
 	return int(C.init_mpu(C.int(sample_rate), C.int(yaw_mix_factor)))
 }
 
+func GyroFusion(toggle uint8) int {
```

Comment:
IMHO, it's cleaner and more straight forward to just expose each C function with its equivalent Go wrapper in this case, e.g. EnableFusion and DisableFusion.